### PR TITLE
https://issues.redhat.com/browse/ACM-11979--more changes for hcp

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/hosted_control_planes/hosted_intro.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 With {mce-short} cluster management, you can deploy {ocp-short} clusters by using two different control plane configurations: standalone or hosted control planes. The standalone configuration uses dedicated virtual machines or physical machines to host the {ocp-short} control plane. With hosted control planes for {ocp-short}, you create control planes as pods on a hosting cluster without the need for dedicated physical machines for each control plane.
 
-Hosted control planes for {ocp-short} are available on bare metal and {ocp-virt}, and as a **Technology Preview** feature on Amazon Web Services (AWS). You can host the control planes on {ocp-short} version 4.14. The hosted control planes feature is enabled by default.
+Hosted control planes for {ocp-short} are available on bare metal, {ocp-virt}, and on Amazon Web Services (AWS). The hosted control planes feature is enabled by default.
```
Comment (xenolinux) [inline clusters/hosted_control_planes/hosted_intro.adoc:6]: 1. HCP is also available on IBM Z, IBM Power, and non bare metal agent machines. Would you please add these platforms too?

And then bullets after line no 40 needs to be updated too. 

2. On the other hand, by removing line no 41-46, we can say something like this on line no 6: 

----

You can configure hosted control planes on the following platforms:

* Platform 1 (text linking to the configuring HCP section for platform 1)
* Platform 2  (text linking to the configuring HCP section for platform 2)
* an so on

----

**Review by xenolinux — COMMENTED** [review]
I added one comment that is probably out of scope for this jira. 

Changes related to the jira - LGTM

**Review by xenolinux — APPROVED** [review]

**Review by rokej — APPROVED** [review]

File: clusters/hosted_control_planes/hosted_intro.adoc (hunk 1 above)
Comment (rokej) [reply clusters/hosted_control_planes/hosted_intro.adoc:6]: @swopebe @xenolinux Will this be added to this doc with a different PR?

Comment (oafischer) [reply clusters/hosted_control_planes/hosted_intro.adoc:6]: I will add it in a different PR @rokej.

Jira: https://issues.redhat.com/browse/ACM-12301

**Pull request merged**